code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -98,22 +98,8 @@
 	pepperspray_protect = TRUE
 	flash_protect = FLASH_PROTECTION_SENSITIVE
 
-/// the key to some of their powers
+/// the key to none of their powers
 /obj/item/organ/brain/shadow
 	name = "shadowling tumor"
 	desc = "Something that was once a brain, before being remolded by a shadowling. It has adapted to the dark, irreversibly."
 	icon = 'icons/obj/medical/organs/shadow_organs.dmi'
-	/// What status effect do we gain while in darkness?
-	var/applied_status = /datum/status_effect/shadow_regeneration
```

Comment:
Should probably also delete `/datum/status_effect/shadow_regeneration` from `buffs.dm` if it is now deprecated.
I think finding a way to maintain the screen alert would be helpful though; because shadowpeople can see in the dark the indicator that you're actually _in_ the dark was useful.

Reply:
It's used by fishes (I didn't know this either until it started screaming at me)